README.md

```diff
@@ -6,11 +6,21 @@
 # Imitation Learning Baseline Implementations
 
 This project aims to provide clean implementations of imitation and reward learning algorithms.
-Currently, we have implementations of Behavioral Cloning, [DAgger](https://arxiv.org/pdf/1011.0686.pdf) (with synthetic examples), density-based reward modeling, [Maximum Causal Entropy Inverse Reinforcement Learning](https://www.cs.cmu.edu/~bziebart/publications/maximum-causal-entropy.pdf), [Adversarial Inverse Reinforcement Learning](https://arxiv.org/abs/1710.11248), [Generative Adversarial Imitation Learning](https://arxiv.org/abs/1606.03476) and [Deep RL from Human Preferences](https://arxiv.org/abs/1706.03741).
+Currently, we have implementations of the algorithms below:
 
-Read [the documentation here](https://imitation.readthedocs.io/en/latest/).
+| Algorithm                                             | Paper                                                                    | API Docs |
```

Comment:
Will move the link to the title as you suggested. Can also add links to the docs. Not sure about collecting data from the paper, I think I could probably do it, but would not want to write in something that is wrong, and it might be more time-efficient that someone who is certain fills it in.